src/components/Header/index.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import {Link} from 'react-router-dom';
+import './style.css';
+
+class Header extends Component{
+    render() {
+        const {userName, onLogout, location} = this.props;
+
+        return (
+            <div className="header">
+                <div className="nav">
+                    <span className="left-link">
+                        <Link to="/">首页</Link>
+                    </span>
+                    {(userName && userName.length > 0) ? (
+                        // Pay attention to the styling name, should use camel format without dash
+                        <span className="user" style="marginRight: 5px">
```

Comment:
For html tag class, should "className" in React.
For styling name that has dash, like "margin-right", should use camel format like "marginRight" in React.